docs/wiki/Whats-new.md

```diff
@@ -38,6 +38,13 @@ This article will be updated as and when changes are made to the above and anyth
 
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
+### October 2023
+
+#### Policy
+
+- Added [this policy definition](https://www.azadvertizer.net/azpolicyadvertizer/d074ddf8-01a5-4b5e-a2b8-964aed452c0a.html) to [this initiative.](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Deny-PublicPaaSEndpoints.html)
```

Comment:
Please put in meaningful descriptions for policy definition and initiative. This is not readable currently, and requires clicking links to understand what we're addressing.